report/1.Introduction/1.IntroductionCtrl.tex

```diff
@@ -8,51 +8,53 @@
+Furthermore, if the object is intelligent in any way, the path might change procedurally.
+%TODO procedurally is a bad word for this, isn't it?
 
 Tracking of airborne objects has multiple different applications.
 
 \subsection{Airports}
-Estimated thirteen thousand "bird strikes" happen each year, which is the event of a bird colliding with a vehicle, where the majority is aircrafts\cite{WikiBirdStrike}\cite{WildlifeStrikeReport}.
-Even though 65\% are estimated to cause little damage, the damages are estimated to cost 1.2 billion dollars each year, world-wide \cite{CostOfBirdstrikes}.
+An estimated thirteen thousand "bird strikes" happen each year, which is the event of a bird colliding with a vehicle, primarily aircrafts\cite{WikiBirdStrike}\cite{WildlifeStrikeReport}.
+%TODO can we get rid of this wikipedia source? it makes me angry
+Although 65\% are estimated to cause somewhat insignificant damage, the damages are estimated to cost 1.2 billion dollars each year, world-wide \cite{CostOfBirdstrikes}.
 The majority of these accidents happen at low altitude, during landing or takeoff\cite{CostOfBirdstrikes}.
 Airports utilize different methods of bird control, including killing, trapping, poison and lights\cite{BirdControlAtAirports}.
-Shooting birds is one of the most common methods, and has been observed to be quite effective, while also reinforcing other methods.
-This both kills a singular bird, but also scares away other birds.
+Shooting birds is one of the most commonly used methods, and has been observed to be quite effective, while also reinforcing other methods.
+This both eliminates the individual bird, but also scares away other birds.
 However, it can be expensive in both salaries and effort.
 
 \subsection{Military}
-In a military setting hitting a moving airborne target is relevant in a defence context, for instance destroying incoming projectiles or shooting down hostile planes.
+In a military setting, hitting a moving airborne target is relevant in a defense context, for instance destroying incoming projectiles or shooting down hostile planes.
 Compared to the airport-case, the problem can arise at a variety of locations, both on vehicles, boats, planes and tanks, but also on stationary military installations.
-The American military developed the Phalanx CIWS, Close-in weapon system, which is a radar-guided 20mm Gatling-style rotary cannon, mounted on a base that moves on two axes\cite{PhalanxCIWS}.
+The American military developed the Phalanx CIWS, a close-in weapon system, which is a radar-guided 20mm Gatling-style rotary cannon, mounted on a base that moves on two axes\cite{PhalanxCIWS}.
 These are mounted on ships and can shoot incoming missiles and other airborne objects deemed a threat.
 The Dutch also developed a similar system called the Goalkeeper CIWS.
-These are both autonomous.
+%TODO The Dutch...... military?
```

Comment:
The Dutch...... military? Government? ISIS?

Reply:
restructure this sentence to say something about the fact that this weapon isn't the only of it's kind as [someone] developed the goalkeeper